src/Translatable/Translatable.php

```diff
@@ -674,6 +674,30 @@ public function scopeOrWhereTranslationLike(Builder $query, $key, $value, $local
         });
     }
 
+    /**
+     * This scope sorts results by the given translation field.
+     *
+     * @param \Illuminate\Database\Eloquent\Builder $query
+     * @param string                                $key
+     * @param string                                $sortmethod
+     *
+     * @return \Illuminate\Database\Eloquent\Builder|static
+     */
+    public function scopeOrderByTranslation(Builder $query, $key, $sortmethod = 'asc')
+    {
+        $translationTable = $this->getTranslationsTable();
+        $localeKey = $this->getLocaleKey();
+        $table = $this->getTable();
+        $keyName = $this->getKeyName();
+
+        return $query->join($translationTable, function($join) use ($translationTable, $localeKey, $table, $keyName) {
+            $join->on($translationTable.'.'.$this->getRelationKey(), '=', $table.'.'.$keyName)
```

Comment:
I don't like the indent and linebreaks in this whole block.

I would like it more:
```php
$query
  ->join(function() {
    $join
      ->on()
      ->where();
  })
  ->orderBy()
  ->select()
  ->with();
```

But this is just me and won't be a blocker.